### PR TITLE
feat: add form banner component

### PR DESCRIPTION
**Pull request by SashaShostyr**
Figma: https://www.figma.com/design/W3hxWj6VZydNaNGJSGxRsV/TableKit-Components-(3.0)?node-id=7315-46545&m=dev

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @tablecheck/tablekit-core@3.0.5-canary.228.9316166211.0
  npm install @tablecheck/tablekit-css@3.0.5-canary.228.9316166211.0
  npm install @tablecheck/tablekit-react@3.0.5-canary.228.9316166211.0
  npm install @tablecheck/tablekit-react-css@3.0.5-canary.228.9316166211.0
  npm install @tablecheck/tablekit-react-datepicker@3.0.5-canary.228.9316166211.0
  npm install @tablecheck/tablekit-react-select@3.0.5-canary.228.9316166211.0
  # or 
  yarn add @tablecheck/tablekit-core@3.0.5-canary.228.9316166211.0
  yarn add @tablecheck/tablekit-css@3.0.5-canary.228.9316166211.0
  yarn add @tablecheck/tablekit-react@3.0.5-canary.228.9316166211.0
  yarn add @tablecheck/tablekit-react-css@3.0.5-canary.228.9316166211.0
  yarn add @tablecheck/tablekit-react-datepicker@3.0.5-canary.228.9316166211.0
  yarn add @tablecheck/tablekit-react-select@3.0.5-canary.228.9316166211.0
  ```
</details>

### CODE REVIEW

File: system/react-css/src/config.tsx [hunk 1]
```diff
@@ -39,18 +39,24 @@ export function configureDefaults(defaults: CoreConfigDefaults): void {
...
 export function getSentimentIcon(
-  variant: 'success' | 'error' | 'neutral' | 'info' | 'warning',
+  variant:
+    | 'success'
+    | 'error'
+    | 'neutral'
+    | 'info'
+    | 'warning'
+    | 'orange'
+    | 'purple',
```
Comment (SimeonC) [inline system/react-css/src/config.tsx:49]: These aren't sentiments, please revert 🙇

File: system/react-css/src/structuredComponents/FormBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+export const FormBanner = React.forwardRef<
+  HTMLDivElement,
+  ComposedProps & React.HTMLAttributes<HTMLDivElement>
+>((props, ref) => {
+  const { icon, title, children, ...passThrough } = props;
+  return (
+    <FormBannerCore {...passThrough} ref={ref}>
+      <FormBannerIconWrapper>
+        {icon ?? getSentimentIcon(passThrough['data-variant'])}
```
Comment (SimeonC) [inline system/react-css/src/structuredComponents/FormBanner.tsx:58]:
```suggestion
        {icon ?? getIcon(passThrough['data-variant'])}
```

```ts
function getIcon(variant: Props['data-variant']): JSX.Element {
  if (variant === 'purple' || variant === 'orange')
    return getSentimentIcon('default');
  return getSentimentIcon(variant);
}
```

**Review by SimeonC — CHANGES_REQUESTED** [review]
Made comments in `react-css`, but please make the same changes to `react` equivalent as wess.

File: system/react-css/src/config.tsx (hunk 1 above)
Comment (SashaShostyr) [reply system/react-css/src/config.tsx]: Updated

File: system/react-css/src/structuredComponents/FormBanner.tsx (hunk 2 above)
Comment (SashaShostyr) [reply system/react-css/src/structuredComponents/FormBanner.tsx]: Updated 🙇

File: system/react-css/src/config.tsx [hunk 3]
```diff
@@ -39,18 +39,17 @@ export function configureDefaults(defaults: CoreConfigDefaults): void {
 }
 
 export function getSentimentIcon(
-  variant: 'success' | 'error' | 'neutral' | 'info' | 'warning',
+  variant: 'success' | 'error' | 'warning' | 'default',
```
Comment (SimeonC) [inline system/react-css/src/config.tsx:42]: If you do this you'll need to fix the Banner stories as well as Alert component.

Comment (SashaShostyr) [reply system/react-css/src/config.tsx]: Fixed

**Review by SimeonC — APPROVED** [review]

**Review by anarakhlina — APPROVED** [review]

**Pull request merged**